processes.py

```diff
@@ -72,10 +72,11 @@ def process_to_dict(process):
 
   ProcessStage(
       'Dev trials and iterate on design',
-      'Publicize test-readiness.  Share sample code. '
+      'Publicize availablity for developers to try. '
+      'Provide sample code. '
       'Request feedback from browser vendors.',
-      ['DevRel publicity request',
-       'Samples',
+      ['Samples',
+       'Draft API overview (may be on MDN)',
```

Comment:
Strike "(may be on MDN)" here, as it's misleading.  (Leave it in the existing feature track.)

Reply:
Done.